lib/ASRouter.jsm

```diff
@@ -524,6 +524,15 @@ class _ASRouter {
     ASRouterPreferences.addListener(this.onPrefChange);
     BookmarkPanelHub.init(this.handleMessageRequest, this.addImpression);
 
+    // If we're in ASR debug mode add the local test providers
+    if (ASRouterPreferences.devtoolsEnabled) {
```

Comment:
I guess this is going to require a restart?

Reply:
Moved that into a function so we can update on pref change as well.